src/prelude.rs

```diff
@@ -10,6 +10,9 @@ pub use crate::declare_types;
 #[cfg(all(not(feature = "napi-1"), feature = "event-handler-api"))]
 pub use crate::event::EventHandler;
 #[cfg(all(feature = "napi-4", feature = "event-queue-api"))]
+pub use crate::event::{Channel, SendError};
+#[cfg(all(feature = "napi-4", feature = "event-queue-api"))]
+#[allow(deprecated)]
```

Comment:
Exposing these in the prelude means that people using `use neon::prelude::*;` won't get deprecation warnings, but it seemed better not to break backwards compatibility. Seem like the right call?

Reply:
Since these are only type aliases and aliases are identical to the type they refer, maybe we can _duplicate_ the aliases here instead of importing them?

Will that cause _all_ users to get the deprecation warning even if they aren't using it?

Reply:
Oh, I was wrong! I did an experiment and it turns out that when you import `use neon::prelude::*;` it only warns when you actually use the deprecated names, so that's actually perfect!

[example](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=3e92e2161229edbc62bbb16936ca34e2)

Reply:
@dherman Did you need to make a change for this?

Reply:
No, what I figured out was that this is perfect:

- the `#[allow(deprecated)]` inside the prelude means that Neon itself builds without warnings
- if anyone does `use neon::prelude::*;` without referencing the deprecated names, they get zero warnings
- if anyone does `use neon::prelude::*;` and references the deprecated names, they get a warning at the place where they reference the deprecated names

So it's exactly what we'd want, the way it is.